packages/react-native-bridge/common/gutenberg-web-single-block/content-functions.js

```diff
@@ -25,19 +25,12 @@ window.getHTMLPostContent = () => {
 };
 
 window.insertBlock = ( blockHTML ) => {
-	const { blockEditorSelect, blockEditorDispatch } =
-		window.getBlockEditorStore();
-
 	// Setup the editor with the inserted block.
 	const post = window.wp.data.select( 'core/editor' ).getCurrentPost();
 	window.wp.data
 		.dispatch( 'core/editor' )
 		.setupEditor( post, { content: blockHTML } );
 
-	// Select the first block.
-	const clientId = blockEditorSelect.getBlocks()[ 0 ].clientId;
```

Comment:
This often threw an error where the first block was `undefined`. This logic was relocated to `editor-behavior-overrides` to await the `iframe` render.